src/AnalyzeView/MAVLinkInspectorController.cc

```diff
@@ -84,7 +84,12 @@ MAVLinkInspectorController::~MAVLinkInspectorController()
 {
     qDeleteAll(_timeScaleSt);
     qDeleteAll(_rangeSt);
-    _charts->clearAndDeleteContents();
+
+    for (int i = _charts->count() - 1; i >= 0; i--) {
+        auto chartObject = _charts->get(i);
+        MAVLinkChartController* chart = dynamic_cast<MAVLinkChartController*>(chartObject);
+        deleteChart(chart);
+    }
```

Comment:
I don't understand how this fixes a problem? As far as I can tell it ends up being functionally the same thing. The only different is that chartsChanged is not signalled. But given the fact this is happening during destruction that should be needed.